scripts/validate_wheel.sh

```diff
@@ -153,7 +155,7 @@ for ex in `find "$root_folder/snippets" -name '*.py'`; do
 done
 
 # Run examples
-for ex in `find "$root_folder/examples" -name '*.py' -not -path '*/providers/*'`; do
+for ex in `find "$root_folder/examples" -name '*.py'`; do
```

Comment:
Do we need to test the `applications` or `targets` directories that were added? If some tests were moved into the those directories, then I think we should add them here so that we don't reduce our test coverage. Happy to discuss if you think otherwise though.